tests/libYARP_math/RandTest.cpp

```diff
@@ -48,11 +48,18 @@ class RandTest : public UnitTest {
         printf("done!\n");
         printf("Now performing some *basic* tests on the sequence\n");
 
+        double min=1e10;
+        double max=0.0;
```

Comment:
Not a big problem in this test code, but in general you can use the std::numeric_limits class for this kind of initializations: http://www.cplusplus.com/reference/limits/numeric_limits/ .